checks/sast.go

```diff
@@ -49,10 +49,16 @@ func SAST(c *checker.CheckRequest) checker.CheckResult {
 		return checker.CreateRuntimeErrorResult(CheckSAST, codeQlErr)
 	}
 
+	tfsecScore, tfsecErr := tfsecCheckDefinitions(c)
```

Comment:
if tfsecCheckDefinitions() succeeds, we're returning early. If repos with such files never have other type of code and don't need to have codeQL enabled, the code works. If it's possible they have both terraform code and other programming languages code, I think this will create false positive, ie high score even though dependabot is not enabled.

Wdut?

Is there a way to tweak the logic to accommodate both scenarios?

Reply:
I think that's a great idea to use that  API 👍